Source/Core/DolphinQt2/GameList/GameList.cpp

```diff
@@ -154,10 +155,24 @@ void GameList::ShowContextMenu(const QPoint&)
   }
   if (platform == DiscIO::Platform::WII_WAD)
   {
-    menu->addAction(tr("Install to the NAND"), this, SLOT(InstallWAD()));
+    QAction* wad_install_action = new QAction(tr("Install to the NAND"), menu);
+    QAction* wad_uninstall_action = new QAction(tr("Uninstall from the NAND"), menu);
 
-    if (GameFile(game).IsInstalled())
-      menu->addAction(tr("Uninstall from the NAND"), this, SLOT(UninstallWAD()));
+    connect(wad_install_action, &QAction::triggered, this, &GameList::InstallWAD);
+    connect(wad_uninstall_action, &QAction::triggered, this, &GameList::UninstallWAD);
+
+    for (QAction* a : {wad_install_action, wad_uninstall_action})
+    {
+      connect(this, &GameList::EmulationStarted, [a] { a->setEnabled(false); });
```

Comment:
I think this should be:

    connect(this, &GameList::EmulationStarted, a, [a] { ... });

to make sure that the lambda will only be called if a exists. And same for below.